python/pyspark/ml/wrapper.py

```diff
@@ -88,6 +99,18 @@ def _transfer_params_from_java(self):
                     value = _java2py(sc, self._java_obj.getOrDefault(java_param))
                     self._paramMap[param] = value
 
+    def _transfer_extra_params_from_java(self, extra):
```

Comment:
Same here: Rename to _transfer_param_map_from_java, and rename "extra" to "java_param_map"